### PR TITLE
Add 120 char column recommendation

### PR DESCRIPTION
**Pull request by brianhlin**
At risk of starting a religious war...

### CODE REVIEW

File: docs/documentation/style-guide.md [hunk 1]
```diff
@@ -47,11 +58,14 @@ For command blocks and file snippets that inside of a list should use the approp
...
 
-We use the [Pygments](http://pygments.org/) highlighting library for syntax; it knows about 100 different languages.  The Pygments website contains a live renderer if you want to see how your text will come out.  Please use the `console` language for shell sessions.
+We use the [Pygments](http://pygments.org/) highlighting library for syntax; it knows about 100 different languages.
+The Pygments website contains a live renderer if you want to see how your text will come out.  Please use the `console`
+language for shell sessions.
 
 ### Root and user prompts ###
 
-When specifying instructions for the command-line, indicate to users whether the commands can be run as root (`root@host # `) or as an unprivileged user (`user@host $ `). 
+When specifying instructions for the command-line, indicate to users whether the commands can be run as root (`root@host
```
Comment (matyasselmeci) [inline docs/documentation/style-guide.md]: Please do not break inside a `code` section.

**Comment by matyasselmeci** [comment]
I'm a big fan of the "one-line-per-sentence" method myself -- makes diffs easier to read.
(/me opens fire)

**Comment by brianhlin** [comment]
How about at most one sentence per line, limited to 120 chars wide?

**Comment by matyasselmeci** [comment]
It should be more like advice than a requirement. Something to the effect of:
> Wherever possible, prose should be limited to 120 characters wide. In addition, using one line for each sentence makes it easier to review diffs in updates.

**Pull request merged**